src/python/grapl-tests-common/grapl_tests_common/setup.py

```diff
@@ -41,26 +35,12 @@ def _upload_analyzers(
 
 
 def _upload_test_data(
-    s3_client: S3ServiceResource, test_data_paths: Sequence[str]
+    s3_client: S3ServiceResource, test_data: Sequence[UploadTestData]
 ) -> None:
-    logging.info(f"Running upload-sysmon-logs")
-
-    # i hate this lol
-    # but it's probably better than mucking with path and importing that module...
```

Comment:
motivation for why I refactored it so that it's provided as a library.

(if we were calling this script from the Grapl AWS Plugins repo, we'd need to also publish the `etc/` directory as a Dockerhub image or something gross. Just providing it all as a py library is much cleaner.)